Rendera/include/cube.hpp

```diff
@@ -0,0 +1,209 @@
+#ifndef CUBE_HPP
+#define CUBE_HPP
+
+#include "vec3.hpp"
+#include "ray.hpp"
+#include "material.hpp"
+
+#define INFINITY 1E10
+
+struct HitInfo
```

Comment:
You shouldn't define the `HitInfo` struct, as that task was assigned to Mahesh. You can assume its existence and write code which uses the struct, however.